### ISSUE TITLE
feature request: `phrase hello slash world` -> `hello/world`

### ISSUE THREAD

**Issue by dwiel** [knausj85/knausj_talon#238]
I'm not sure what the best way to go about this is, it might be simple.

some more examples/unit tests:

- `phrase hello slash world` -> `hello/world` (currently `hello /world`)
- `phrase hello slash snake name of file slash world` -> `hello/name_of_file/world` (currently `hello/name_of_file_/world`)

Similarly: `phrase file dot extension` -> `file.extension`, though this one might be complicated by the fact that I have a formatter called `dot` defined so that `dot hello world` -> `.hello_world` because I use python so much.

**Comment by rntz** [comment]
This is not resolved yet, but #243 should show the way towards implementing this.

**Comment by rntz** [comment]
If you still want this behavior, the easiest way to do it is to add a formatter in formatters.py that's mapped to NOOP. I have one of these and I call it `type`:

```
# This is the mapping from spoken phrases to formatters
formatters_words = {
...
    "type": formatters_dict["NOOP"],
...
}
```

Then your examples become:

- `type hello slash world` -> `hello/world`
- `type hello slash snake name of file slash world` -> `hello/name_of_file/world`

You could also call this formatter `phrase` and remove or rename the existing phrase command if you wanted phrase to behave like this. I think it's probably more useful to have `phrase` always produce words by default, so I'm going to close this, but you can reopen if you disagree/want to discuss further.
